src/pages/myGeotab/introduction.tsx

```diff
@@ -73,29 +76,30 @@ const integratingWithMyG: ReactNode = (
             <li>Adding and removing a user</li>
         </ul>
         Each of the MyGeotab pages has a URL (Uniform Resource Locator) associated with the page that allows that page to be embedded in your own application or linked to.
-        This is a great way to leverage MyGeotab and make it part of your services. See <a href="../guides/mygeotab-urls/">mygeotab-urls</a> for more details. {/*TODO: fix link*/}
+        This is a great way to leverage MyGeotab and make it part of your services. See <Link to="/myGeotab/guides/MyGeotabUrls">mygeotab-urls</Link> for more details.
```

Comment:
Should we rename this link to the actual page title?
```suggestion
        This is a great way to leverage MyGeotab and make it part of your services. See <Link to="/myGeotab/guides/MyGeotabUrls">Using MyGeotab URLs</Link> for more details.
```